SEOImageUpdates: User Story 1463880, Small PR 2 (#12338)
dotnet/docs: docs/framework/windows-workflow-foundation/state-machine-workflows.md

  
 To create a state machine workflow, states are added to a <xref:System.Activities.Statements.StateMachine> activity, and transitions are used control the flow between states. The following screenshot, from the [Getting Started Tutorial](getting-started-tutorial.md) step [How to: Create a State Machine Workflow](how-to-create-a-state-machine-workflow.md), shows a state machine workflow with three states and three transitions. **Initialize Target** is the initial state and represents the first state in the workflow. This is designated by the line leading to it from the **Start** node. The final state in the workflow is named **FinalState**, and represents the point at which the workflow is completed.  
  

>  ![Completed state machine workflow from the Getting Started Tutorial](./media/state-machine-workflows/getting-started-tutorial-complete-state-machine-workflow.jpg)  

Reviewer: ```suggestion
 ![Illustration that shows the completed state machine workflow.](./media/state-machine-workflows/complete-state-machine-workflow.jpg)  
```

  
 A state machine workflow must have one and only one initial state, and at least one final state. Each state that is not a final state must have at least one transition. The following sections cover creating and configuring states and transitions.  
  
## Creating and Configuring States  
 A <xref:System.Activities.Statements.State> represents a state in which a state machine can be in. To add a <xref:System.Activities.Statements.State> to a workflow, drag the **State** activity designer from the **State Machine** section of the **Toolbox** and drop it onto a <xref:System.Activities.Statements.StateMachine> activity on the [!INCLUDE[wfd1](../../../includes/wfd1-md.md)] surface.  
  

>  ![WF4 State Machine Activities](./media/state-machine-workflows/net-framework-platform-update-state-machine-activities.jpg)  

Reviewer: ```suggestion
 ![Screenshot of the State Machine section of the Toolbox](./media/state-machine-workflows/state-machine-section-toolbox.jpg)  
```
